pkg/jobrunaggregator/jobrunaggregatorapi/types_row_testrun.go

```diff
@@ -0,0 +1,98 @@
+package jobrunaggregatorapi
+
+import (
+	"strings"
+
+	"cloud.google.com/go/bigquery"
+
+	prowv1 "k8s.io/test-infra/prow/apis/prowjobs/v1"
+
+	"github.com/openshift/ci-tools/pkg/junit"
+)
+
+const (
+	TestRunTableName = "TestRuns"
+
+	// The TestRunsSchema below is used to build the "TestRuns" table.
+	//
+	TestRunsSchema = `
+[
+  {
+    "mode": "REQUIRED",
+    "name": "Name",
+    "description" : "Name of the test run",
+    "type": "STRING"
+  },
+  {
+    "mode": "REQUIRED",
+    "name": "JobRunName",
+    "description" : "Name of the JobRun (big number) that ran this test (e.g., 1389486541524439040)",
+    "type": "STRING"
+  },
+  {
+    "mode": "REQUIRED",
+    "name": "JobName",
+    "description" : "Name of the Job that as this test in it",
+    "type": "STRING"
+  },
+  {
+    "mode": "REQUIRED",
+    "name": "Status",
+    "description" : "Status of the test (e.g., pass, fail)",
+    "type": "STRING"
+  },
+  {
+    "mode": "NULLABLE",
+    "name": "TestSuite",
+    "description" : "Testsuite that this test belongs to",
+    "type": "STRING"
+  }
+]
+`
+)
+
+// Move here from jobrunbigqueryloader/types.go
+//
+type TestRunRow struct {
+	ProwJob    *prowv1.ProwJob
+	JobRun     JobRunInfo
+	TestSuites []string
+	TestCase   *junit.TestCase
+}
+
+// Ensure (at compile time) that testRunRow implements the bigquery.ValueSaver interface
+var _ bigquery.ValueSaver = &TestRunRow{}
+
+func (v *TestRunRow) Save() (map[string]bigquery.Value, string, error) {
```

Comment:
Avoid using our API types as method receivers. It seems convenient, but it causes deeper package dependencies for APIs we want to rely on elsewhere and historically has caused diamond dependency resolution problems.